cn.lpkt.serverbox.app.desktop

```
@@ -0,0 +1,6 @@
+[Desktop Entry]
+Name=ServerBox
+Exec=/app/AppRun
```

```suggestion
Exec=/app/AppRun
```

install the executable to `/app/bin` then drop the path from here

---

I think bbhtt meant:

```suggestion
Exec=AppRun
```